### PR TITLE
USM: tests: eliminate sleeps

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

The PR contains multiple changes in USM UTs, to eliminate tests and cut the total runtime.
Each changed is scoped within its own commit.

Total saving time ~1m

### Motivation

Cut kitchen tests time

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/network/tracer/testutil/prefetch_file/prefetch_file.go
Comment (hmahmood) [inline pkg/network/tracer/testutil/prefetch_file/prefetch_file.go:11]: Is this file only used for USM tests? Can the ownership be changes to USM?

Comment (guyarb) [reply pkg/network/tracer/testutil/prefetch_file/prefetch_file.go:11]: yes, I have a dart for that (waiting for the code freeze to end)

Comment (hmahmood) [reply pkg/network/tracer/testutil/prefetch_file/prefetch_file.go:11]: You can add the codeowners change to this PR and that way this PR is not required by networks to review?

Comment (guyarb) [reply pkg/network/tracer/testutil/prefetch_file/prefetch_file.go:11]: Unfortunately no, each PR relies on the CODEOWNERS in `main` branch do determine the reviewers
Once the code freeze will end, I'll merge the CODEOWNERS change, and then merge this
that way we won't need an approval on that file

Comment (guyarb) [reply pkg/network/tracer/testutil/prefetch_file/prefetch_file.go:11]: https://github.com/DataDog/datadog-agent/pull/19891

**Review by hmahmood — APPROVED** [review]

**Review by DanielLavie — CHANGES_REQUESTED** [review]

**Review by DanielLavie — APPROVED** [review]

**Pull request merged**
